msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -588,18 +492,18 @@ class NativeAuthPublicClientApplication(
             }
 
             val parameters =
-                CommandParametersAdapter.createSignUpStartUsingPasswordCommandParameters(
+                CommandParametersAdapter.createSignUpStartCommandParameters(
                     nativeAuthConfig,
                     nativeAuthConfig.oAuth2TokenCache,
                     username,
                     password,
-                    attributes?.toMap()
+                    attributes?.userAttributes
```

Comment:
why this change?

Reply:
This is what it is on line 781 in the previous version of the file.

Reply:
It seems we had an inconsistency between `signUp` and `signUpUsingPassword` when it comes to this field.
Could you validate that `attributes?.userAttributes` is the right way of doing this, and not `attributes?.toMap()`, by doing some E2E testing.

Reply:
`SignUpStates.submitAttribute` still uses `toMap()`, so this inconsistency needs to be fixed. Let's make sure we're not introducing a last minute bug in this flow.